tests/test_installer.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2024 Science project contributors.
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import subprocess
+from pathlib import Path
+
+import pytest
+from _pytest.tmpdir import TempPathFactory
+
+
+@pytest.fixture(scope="module")
+def installer(build_root: Path) -> Path:
+    return build_root / "install.sh"
+
+
+def run_captured(cmd: list):
+    return subprocess.run(cmd, capture_output=True, text=True)
```

Comment:
This is failing on Windows because Windows does not do shebangs. The scie-jump tests don't have this problem because the script is run directly by the job step (which runs in bash): https://github.com/a-scie/jump/blob/83bfd085d428f5c0746acdab45d7bb401aba28f8/.github/workflows/ci.yml#L58

I think you'll need args: `["bash", script, ...]`.

Reply:
Sent you an invite so you can self-serve run /  retry CI.

Reply:
👍

Reply:
Yeah, so you're getting the wsl bash via PATH, it looks like you need the git bash. I think plumbing an env var from the CI step that captures the current active bash the runner step is using (which is the right bash) is needed.